### DISCUSSION TITLE
Dudas sobre GitHub campus experts

### DISCUSSION THREAD

**Discussion by lasr21** [gittogethers/octogatosconf22#172] [Talks in Spanish]
Hola, puedes escribir tus dudas sobre el programa

**Comment by josephinoo** [comment]
@lasr21 no recibo el correo que se puede hacer.Saludos desde Ecuador

**Reply by lasr21** [reply]
> @lasr21 no recibo el correo que se puede hacer.Saludos desde Ecuador

Holaa, lo reviso

**Reply by lasr21** [reply]
> @lasr21 no recibo el correo que se puede hacer.Saludos desde Ecuador

Hola me puedes escribir por correo lasr21 [at] github.com
